### ISSUE TITLE
phate threads should allow ALL

### ISSUE THREAD

**Issue by linsalrob** [carolzhou/multiPhATE2#7]
Per the [documentation](https://github.com/carolzhou/multiPhATE2#running-phate-as-an-embarassingly-parallel-code) you should be able to set

```
phate_threads='ALL'
```

in the config file. This gives me this error:

```
Traceback (most recent call last):
  File "/home/edwa0468/multiphate2/multiPhATE2/multiPhate.py", line 2083, in <module>
    if int(phateThreads) == 0:
ValueError: invalid literal for int() with base 10: 'ALL'
```

**Comment by jeffkimbrel** [comment]
Thanks Rob, someone else also found this yesterday (#5). I thought it was as simple as the error suggests (trying to convert 'ALL' to an integer), but it may be a little more than that.

**Comment by carolzhou** [comment]
It was indeed as simple as that. An "int" check needed to be included, as I added an option to turn off phate threading altogether. "ALL" is accounted for. The number of threads allowed does not exceed os.cpu_count() or the number of genomes being processed.

**Comment by linsalrob** [comment]
This has been closed by commit [2ba6709](https://github.com/carolzhou/multiPhATE2/commit/2ba6709de3a8572187acb7dae34adcbe320cc094)
